paddle/fluid/operators/mkldnn/lrn_mkldnn_op.cc

```diff
@@ -27,10 +27,12 @@ class LRNMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
  public:
   void Compute(const paddle::framework::ExecutionContext& ctx) const override {
     const bool is_float_type = std::is_same<T, float>::value;
-    PADDLE_ENFORCE(is_float_type, "MKLDNN LRN must use float data.");
-    PADDLE_ENFORCE(paddle::platform::is_cpu_place(ctx.GetPlace()),
-                   "MKLDNN LRN must use CPUPlace.");
-
+    PADDLE_ENFORCE_EQ(
+        is_float_type, true,
+        platform::errors::PreconditionNotMet("DNNL LRN must use float data."));
```

Comment:
It is better to print the type of data in this error message.

Reply:
How about print the type of data in the next PR (macro `GET_PLACE_CPU_CHECK`)? @GaoWei8

Reply:
> How about print the type of data in the next PR (macro `GET_PLACE_CPU_CHECK`)? @GaoWei8

Sounds good. This PR can be merged.